Can't generate metadata for Chainflip

This used to work:
`npx polkadot-api add -w "wss://mainnet-archive.chainflip.io/" chainflip`
Though now it returns this error:
```
Replacing existing chainflip config
✔ Metadata saved as .papi/metadata/chainflip.scale
Saved new spec "chainflip"
Reading metadata
Generating descriptors
CLI Building entry: .papi/descriptors/src/index.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v8.3.6
CLI Target: esnext
CJS Build start
ESM Build start
✘ [ERROR] Unexpected "*"

    .papi/descriptors/src/chainflip.ts:3872:1:
      3872 │  *can be continued and sets the Assethub Pure Proxy Vault in environment pallet. The
      
```

--- a/packages/cli/CHANGELOG.md
+++ b/packages/cli/CHANGELOG.md
@@ -5,6 +5,7 @@
 ### Fixed
 
 - Ensure final newline is added on all files
+- Fix entry docs not getting stripped properly [#1033](https://github.com/polkadot-api/polkadot-api/issues/1033).
 
 ## 0.11.13 - 2025-05-05
 

--- a/packages/client/CHANGELOG.md
+++ b/packages/client/CHANGELOG.md
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+### Fixed
+
+- **CLI:**
+  - Fix entry docs not getting stripped properly [#1033](https://github.com/polkadot-api/polkadot-api/issues/1033).
+
 ## 1.10.2 - 2025-05-05
 
 ### Fixed

--- a/packages/codegen/CHANGELOG.md
+++ b/packages/codegen/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Fixed
+
+- fix entry docs not getting stripped properly [#1033](https://github.com/polkadot-api/polkadot-api/issues/1033).
+
 ## 0.14.0 - 2025-04-24
 
 ### Changed

--- a/packages/codegen/src/generate-descriptors.ts
+++ b/packages/codegen/src/generate-descriptors.ts
@@ -27,7 +27,7 @@ export const customStringifyObject = (
     .map(([key, value]) => {
       if (isDocs(value)) {
         return `\n\n/**\n${value.docs
-          .map((doc: string) => ` *${doc}`)
+          .map((doc: string) => ` * ${doc.trim()}`)
           .join("\n")}\n */\n${key}: ${customStringifyObject(value.value)}`
       }
       return `${key}: ${value}`

--- a/packages/codegen/src/generate-docs-descriptors.ts
+++ b/packages/codegen/src/generate-docs-descriptors.ts
@@ -359,7 +359,7 @@ function buildTypeFolder(
       docs.length > 0
         ? `
 /**
-${docs.map((doc: string) => ` *${doc}`).join("\n")}
+${docs.map((doc: string) => ` * ${doc.trim()}`).join("\n")}
  * @packageDocumentation
  */\n\n
 `
@@ -393,7 +393,7 @@ export {${section}_${typeName} as ${typeName}};`,
 
 function formatDocs(docs: string[]): string {
   return `/**
-${docs.map((doc: string) => ` *${doc}`).join("\n")}
+${docs.map((doc: string) => ` * ${doc.trim()}`).join("\n")}
 */
 `
 }

--- a/packages/codegen/src/internal-types/generate-typescript.ts
+++ b/packages/codegen/src/internal-types/generate-typescript.ts
@@ -156,7 +156,7 @@ export const generateObjectCode = (
     code: `{${innerValues
       .map(({ label, docs, value, result }) => {
         const docsPrefix = docs.length
-          ? `\n/**\n${docs.map((doc) => ` *${doc}`).join("\n")}\n */\n`
+          ? `\n/**\n${docs.map((doc) => ` * ${doc.trim()}`).join("\n")}\n */\n`
           : ""
         if (result === null)
           return docsPrefix + `${JSON.stringify(label)}: undefined`

